app/styles/components/sy-toggle.scss

```diff
@@ -5,7 +5,7 @@
   margin: auto;
 
   &.active {
-    color: $color-primary;
+    color: rgb(0 102 255);
```

Comment:
Could we try to use `$color-primary` here and the `darken` scss function? This way we are more flexible if we decide to adapt the primary color.

Reply:
now it's using the `darken` function, and it's 95% the same results